src/Panel/Model.php

```diff
@@ -116,7 +128,7 @@ public function image(
 		$settings = array_merge(
 			$this->imageDefaults(),
 			$settings ?? [],
-			$this->model->blueprint()->image() ?? [],
+			$blueprint ?? [],
```

Comment:
A possible shorter fix would be to discard all falsy values with:

```php
$this->model->blueprint()->image() ?: [],
```

Reply:
But then `image: false` wouldn't have any effect at all, would it? I think the intended logic would be that if no settings are set and blueprint says false, return null (no image). And not to use the defaults.

Reply:
I think it depends if you want to consider that `image: false` is incorrect input that should be ignored, or a new feature you want to add.

(Personally I don't think I have a use case for `image: false` disabling the default image, at least not currently. I did have `image: false` in one of my blueprints, but I think that was just leftover cruft from some trial-and-error when originally writing those blueprints with Kirby 3.9.)

Reply:
I think it would be more consistent in the handling, where you can define a general default in the model blueprint in general but this can be overwritten by the option in the section definition.